about.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Visit Uganda Summit</title>
+    <link rel="stylesheet" href="about.css">
+    <link href='https://fonts.googleapis.com/css?family=lato' rel='stylesheet'>
+</head>
+<body>
+    <div class="mobile-menu" id="mobile-menu">
+        <div class="mobile-menu-header">
+          <button type="button" class="close-button">&times;</button>
+        </div>
+        <nav>
+          <a href="./index.html" class="navigate">
+            Home
+          </a>
+          <a href="./about.html" class="navigate">
+            About
+          </a>
+          <a href="./index.html#program" class="navigate">
+            Program
+          </a>
+          <a href="#" class="navigate">
+            Join
+          </a>
+          <a href="./about.html#sponsor" class="navigate">
+            Sponsor
+          </a>
+        </nav>
+      </div>
+      <header>
+        <div class="social-media">
+          <nav class="social-nav">
+            <a href="https://facebook.com/edith.naisanga"><img src="./images/facebook-icon.png" alt="facebook"></a>
+            <a href="https://twitter.com/edithnaisanga"><img src="./images/twitter-icon.png" alt="twitter"></a>
+            <a href="#">English</a>
+            <a href="https://github.com/nedith">My page</a>
+            <a href="#">Logout</a>
+          </nav>
+        </div>
+        <div class="navbar">
+          <div class="navbar-logo-div"><a href="./index.html"><img class="summit-logo" src="./images/logo.jpg" alt=""></a></div>
```

Comment:
Please add an alternative text to this image.

<img width="837" alt="Screen Shot 2022-06-22 at 9 22 13" src="https://user-images.githubusercontent.com/82426682/175053117-e8c71e19-aaaa-4e6b-a241-dea8b98d4e2e.png">